packages/edge/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@adobe/react-native-aepedge",
+  "version": "1.0.0-alpha.0",
+  "description": "Adobe Experience Platform support for React Native apps.",
+  "homepage": "https://aep-sdks.gitbook.io/docs/",
+  "license": "Apache-2.0",
+  "main": "./js/index.js",
+  "scripts": {
+    "cleanup": "rm -rf node_modules",
+    "flow": "flow"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/adobe/aepsdk-react-native",
+    "directory": "packages/edge"
+  },
+  "keywords": [
+    "react-native",
+    "adobe",
+    "adobe experience platform",
+    "edge",
+    "ios",
+    "android"
+  ],
+  "author": {
+    "name": "Adobe Experience Platform SDK Team"
+  },
+  "peerDependencies": {
+    "@adobe/react-native-aepcore": "1.0.0",
```

Comment:
Can we use a version range for Core -> `"@adobe/react-native-aepcore": "^1.0.0"`